appinventor/components/src/com/google/appinventor/components/runtime/PhoneCall.java

```diff
@@ -56,9 +59,11 @@
     nonVisible = true,
     iconName = "images/phoneCall.png")
 @SimpleObject
-@UsesPermissions(permissionNames = "android.permission.CALL_PHONE, android.permission.READ_PHONE_STATE, android.permission.PROCESS_OUTGOING_CALLS")
-public class PhoneCall extends AndroidNonvisibleComponent implements Component, OnDestroyListener {
+@UsesPermissions(Manifest.permission.READ_PHONE_STATE)
+public class PhoneCall extends AndroidNonvisibleComponent implements Component, OnDestroyListener, ActivityResultListener {
 
+  private static final String LOG_TAG = PhoneCall.class.getSimpleName();
+  private static final int REQUEST_CODE = 0x50484F4E;
```

Comment:
@ewpatton Where does this value come from? At least put in a comment so we know...